CIP-0036/README.md

```diff
@@ -230,6 +230,10 @@ Fund 8:
  - added the `voting_purpose` field to limit the scope of the delegations.
  - rename the `staking_pub_key` field to `stake_credential` and `registration_signature` to `registration_witness` to allow for future credentials additions.
 
+Fund 10:
+- stipulated that `reward_address` must be a Shelley payment address, otherwise voting reward payments will not be recieved. 
+  - **Note:** up to Fund 9 `reward_address` was a Shelley rewards address; from Fund 10 onwards, it will be a normal payment address.  This will allow rewards to be paid directly from the designated Catalyst pot rather than requiring MIR transfers.
```

Comment:
```suggestion
  - **Note:** up to Fund 9 `reward_address` was a Shelley reward address (a.k.a. stake address); from Fund 10 onwards, it will be a normal payment address.  This will allow rewards to be paid directly from the designated Catalyst pot rather than requiring MIR transfers.
```